Source/FakeItEasy/Core/DefaultFakeWrapperConfigurer.cs

```diff
@@ -6,21 +6,27 @@ namespace FakeItEasy.Core
     /// Handles configuring of fake objects to delegate all their calls to a wrapped instance.
     /// </summary>
     internal class DefaultFakeWrapperConfigurer
-        : IFakeWrapperConfigurer

```

Comment:
Given that the removal of the interface means that an alternate implementation can no longer be used anywhere, I suggest this class is renamed to `FakeWrapperConfigurer`.